app/models/provider_user.rb

```diff
@@ -61,15 +61,6 @@ def can_manage_organisations?
     provider_permissions.exists?(manage_organisations: true)
   end
 
-  def primary_provider_type
```

Comment:
could we also remove the call to this method from https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/controllers/provider_interface/provider_interface_controller.rb#L37 as it is causing the following error: https://sentry.io/organizations/dfe-teacher-services/issues/3148141221/?project=1765973&referrer=slack

Reply:
@cwrw Apologies this wasn't removed in the commit I expected. Rebased now https://github.com/DFE-Digital/apply-for-teacher-training/pull/7021/commits/d97fce7befa4053ddcc57d305f6e2acbf860609a#diff-5183cb85f5b7e77048cc9ae789796cfa9ae0378ac7743b6f7a8294a4bf6fa35fL37